napari/layers/base/base.py

```diff
@@ -732,3 +731,35 @@ def on_mouse_release(self, event):
         """Called whenever mouse released in canvas.
         """
         return
+
+
+def convert_to_uint8(data: np.ndarray):
```

Comment:
I would put this somewhere in utils and it needs a docstring.

Reply:
done